sites/funmooc/src/backend/templates/richie/base.html

```diff
@@ -20,3 +20,14 @@
 {% endblock topbar_contact %}
 
 {% block body_footer_title %}{% endblock body_footer_title %}
+{% block body_js %}
+    <script type="text/javascript">
+        window.__funmooc_context__ = {
+            privacy: JSON.parse('{{ PRIVACY_CONTEXT|safe }}'),
+            marketing: JSON.parse('{{ MARKETING_CONTEXT|safe }}'),
+        }
+    </script>
+    <script type="text/javascript" src="{% static 'richie/js/tarteaucitronjs/tarteaucitron.js' %}"></script>
+    <script type="text/javascript" src="{% static 'funmooc/js/tarteaucitron.js' %}"></script>
```

Comment:
duplicate?

Reply:
Ah nope `'richie/js/tarteaucitronjs/tarteaucitron.js'` is the library itself.
`'funmooc/js/tarteaucitron.js'` is the script to initialize tarteaucitron for funmooc.